tests/sentry/web/frontend/test_accounts.py

```diff
@@ -274,6 +297,78 @@ def test_relocate_recovery_invalid_password(self):
                 assert resp.status_code == 200
                 assert resp[header_name] == "strict-origin-when-cross-origin"
 
+    def test_relocate_reclaim_success(self):
+        user = self.create_user(email="member@example.com", is_unclaimed=True)
+        lost_password = LostPasswordHash.objects.create(user=user)
+        org = self.create_organization(name="test-org")
+        self.create_member(user=user, organization=org, role="member", teams=[])
+
+        with patch.object(LostPasswordHash, "send_relocate_account_email") as mock_relocation_email:
```

Comment:
You could read the LostPasswordHash out of the database, and check `django.core.mail.outbox` to see the mail that would have been sent instead of using mocks.

Reply:
Had to use the `BurstTaskRunner` since the email send is async, but otherwise this worked well and actually caught a bug.